test/FPIS_TEST/LoginTest.cs

```diff
@@ -7,26 +7,45 @@ namespace FPIS_TEST
     [TestClass]
     public class LoginTest
     {
-        Login obj;
-        public LoginTest()
+        Login login;
+        [TestInitialize]
+        public void InitializeTest()
         {
-            obj = new Login();
+            login = new Login();
         }
+
         [TestMethod]
-        public void TestTestUserProvidedData()
+        public void TestUserProvidedData()
         {
-            string userInput = "Kofi";
-            bool testResultIsTrue = obj.TestUserProvidedData(userInput);
-            Assert.IsTrue(testResultIsTrue);
+            string userInput = "";
+
+            bool expected = login.UserProvidedData(userInput);
+
+            Assert.AreEqual(expected, false);
         }
 
         [TestMethod]
         public void TestAuthenticateUser()
```

Comment:
What awesomeness are we about to see? Can you remove this method if it's not yet implemented?